src/Cmfd.cpp

```diff
@@ -1551,31 +1161,29 @@ void Cmfd::splitCorners() {
 
 
 /**
- * @brief Get the ID of the Mesh cell next to given Mesh cell.
- * @param current Mesh cell ID
- * @param CMFD cell surface ID to look across for neighboring cell
- * @return neighboring CMFD cell ID
+ * @brief Get the ID of the Mesh cell next to a given Mesh cell across a
+ *        given surface.
+ * @param cell_num Current Mesh cell ID
+ * @param surface_id CMFD cell surface ID to look across for neighboring cell
+ * @return Neighboring CMFD cell ID
  */
 int Cmfd::getCellNext(int cell_num, int surface_id) {
 
   int cell_next = -1;
 
-  if (surface_id == 0) {
+  if (surface_id == SURFACE_X_MIN) {
     if (cell_num % _num_x != 0)
       cell_next = cell_num - 1;
   }
-
-  else if (surface_id == 1) {
+  else if (surface_id == SURFACE_Y_MIN) {
     if (cell_num / _num_x != 0)
       cell_next = cell_num - _num_x;
   }
-
-  else if (surface_id == 2) {
+  else if (surface_id == SURFACE_X_MAX) {
     if (cell_num % _num_x != _num_x - 1)
       cell_next = cell_num + 1;
   }
-
-  else if (surface_id == 3) {
+  else if (surface_id == SURFACE_Y_MAX) {
```

Comment:
Ditto with these ... any way we could determine this based on surface number ... such as having an array contain [_num_x, _num_y, _num_z]